Classes/Domain/Model/Category.php

```diff
@@ -20,16 +21,12 @@
  */
 class Category extends AbstractEntity
 {
-    /**
-     * @Extbase\Validate("NotEmpty")
-     */
+    #[Extbase\Validate(['validator' => 'NotEmpty'])]
     protected string $title = '';
 
     protected string $description = '';
 
-    /**
-     * @Extbase\ORM\Lazy
-     */
+    #[Lazy]
     protected ?Category $parent;
```

Comment:
Please add null as default to prevent uninitialized property error on PHP >= 8.0

Reply:
Done ☑️